### ISSUE TITLE
ToTem

### ISSUE THREAD

**Issue by lrl-hub** [agnivsen/ToTem-NRSfM#1]
Is there any paper 8.5 running code？Please...

**Comment by agnivsen** [comment]
How do you mean by paper 8.5? Could you please elaborate? Are you referring to section 8.5?

**Comment by lrl-hub** [comment]
Thanks for your reply！！I'm talking about section 8.5 in your paper.The photo above shows it.I didn't find the code mentioned in the 8.5 experiment summary in the code you gave.


200521431刘瑞亮
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "agnivsen/ToTem-NRSfM"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月8日(星期五) 下午5:13
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [agnivsen/ToTem-NRSfM] ToTem (Issue #1)





 
How do you mean by paper 8.5? Could you please elaborate? Are you referring to section 8.5?

**Comment by agnivsen** [comment]
You are right, the code does not include a ready script to replicate all figures of section 8.5. I can find the additional scripts for this if you need, but I will need some time to do so. Let me know if you need this, I can get back to you by next week if you really want.

But in the meantime, if you wish, you can do this all by yourself; I will elaborate how:

1. The methodology is already given in <SurfaceBasedNonRigidBundleAdjustment.m>, you just need to change the data to an appropriate one

2.  For the planar template, three data are used: Hulk, KinectPaper, and WhiteCartoonTshort. All the three data are already given in </ToTem-NRSfM/Data/>, you just need to modify <SurfaceBasedNonRigidBundleAdjustment.m> after Line 52 to read either <data_hulk.mat>, <data_kinectPaper.mat>, or <data_whiteCartoonTshirt.mat> into the 'DataSubsampled" variable and <intrinsics_hulk.txt>, <data_whiteCartoonTshirt.mat>, or <data_kinectPaper.mat> into the 'K' variable. If you are unsure how to do this, check Lines 44-45 of <IsometricNRSfM.m> for reference

3. For non-planar topology, the Leggings an Balloon dataset are available at: https://encov.ip.uca.fr/ab/code_and_datasets/index.php. The EndoMapper dataset is in: https://paperswithcode.com/dataset/endomapper, the exact sequence we used is not publicly available, but I can find it out if you need (this can take a while)

4. The weights in Lines 41-43 of <SurfaceBasedNonRigidBundleAdjustment.m> might need some tuning for optimal results

Finally, if you agree, please resolve this issue since this is not a bug. You can still contact me at: i.agniva@gmail.com
